### DISCUSSION TITLE
Disable Unhandled Errors and general TypeErrors when test files and test suites succeed

### DISCUSSION THREAD

**Discussion by bluedistro** [vitest-dev/vitest#5410] [General]
Hi, I want to know if it is possible to ignore all general type errors and unhandled rejection errors that Vitest report/complain about after running the tests. Currently my test is failing on gitlab pipeline because of these errors even though all tests pass. 

The issue is I am migrating a huge code base with over 14000 test suites so it is not possible to resolve all these errors now so it will be helpful to at least get the migration done by muting/ignoring these errors for now and later attend to them.

Thanks in advance for your help!

**Comment by sheremet-va** [comment]
https://vitest.dev/config/#dangerouslyignoreunhandlederrors

It will not fail the test, but it will still report the error. There is no way to hide the errors.

**Reply by bluedistro** [reply]
Thanks very much for the quick reply! I really appreciate it. Do you have any idea then why all tests pass but pipelines still fail when I run the tests in a CI environment ? or any clues to what might be causing that ?

BTW, I had already experimented with [dangerouslyignoreunhandlederrors](https://vitest.dev/config/#dangerouslyignoreunhandlederrors) and it was still  logging errors instead of hiding them

**Reply by bluedistro** [reply]
I 've noticed that the errors are detected by my CI and automatically fails the test when there are errors so I have to find a way to ignore them completely and let the CI exit with 0 when there are errors

**Reply by reggie3-braingu** [reply]
@bluedistro , were you able to find a resolution to making this work in CI?
